### PR TITLE
issue #3513

### PR DESCRIPTION
**Pull request by gperconti**
Possible solution for issue #3513. it's my first collaboration, please say me if need modify anything

### CODE REVIEW

**Comment by alexlarsson** [comment]
We use g_log() and g_warning() for internal unexpected errors, for errors that are caused by user input and normally displayed, instead use something like:
```
   g_printerr (_("Warning: ....\n");
```

Also, if (like the most common case) you only specify one argument, I think we should probably make that an error if it doesn't exist.

**Comment by gperconti** [comment]
Ready, Change g_log to g_printerr so that it displays a warning message

**Comment by gperconti** [comment]
@alexlarsson  I made the suggested changes, it throws error if a single argument is passed, otherwise it throws warning for each one that is not installed

File: app/flatpak-builtins-uninstall.c [hunk 1]
```diff
@@ -307,12 +307,19 @@ flatpak_builtin_uninstall (int argc, char **argv, GCancellable *cancellable, GEr
...
+                g_set_error (error, FLATPAK_ERROR, FLATPAK_ERROR_NOT_INSTALLED,
                            _("%s/%s/%s not installed"),
                            id ? id : "*unspecified*",
                            arch ? arch : "*unspecified*",
                            branch ? branch : "*unspecified*");
-              return FALSE;
+                return FALSE;
+              }
+              
+              g_printerr("Warning: %s is not installed\n", pref);
```
Comment (alexlarsson) [inline app/flatpak-builtins-uninstall.c:320]: This needs to be translated too, so add _() around the string. And also, there is a missing space before the first parenthesis

**Comment by alexlarsson** [comment]
This is failing the tests:
```
++ flatpak --system uninstall -y org.test.Platform org.test.Hello
ok flatpak list --arch --columns works
PASS: tests/test-repo@user.wrap 21 flatpak list --arch --columns works
Warning: org.test.Platform is not installed
Warning: org.test.Hello is not installed
++ assert_not_reached 'Should not be able to uninstall --system when installed --user'
Should not be able to uninstall --system when installed --user
```

I guess we need to change that `uninstall -y org.test.Platform org.test.Hello` to just uninstall one ref in order to make the tests not fail (it wants an error uninstalling something that is not there). This is in tests/test-repo.sh around line 463.

**Comment by alexlarsson** [comment]
I squashed all the changes into one commit and added a fix for the tests.

**Pull request merged**